cmd/cmd.go

```diff
@@ -937,6 +1032,53 @@ func RunServer(cmd *cobra.Command, _ []string) error {
 	return server.Serve(ln, origins)
 }
 
+func getImageData(filePath string) ([]byte, error) {
+	file, err := os.Open(filePath)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	buf := make([]byte, 512)
+	n, err := file.Read(buf)
+	if err != nil {
+		return nil, err
+	}
+
+	if n != 512 {
+		return nil, fmt.Errorf("invalid image type")
+	}
```

Comment:
This is incorrect. http.DetectContentType consume _at most_ 512 bytes but it doesn't require 512 bytes to detect. The jpg header is only 3 bytes (0xff 0xd8 0xff)